requirements.txt

```diff
@@ -0,0 +1,2 @@
+pygame >= 1.9.2
+numpy >= 1.12.1
```

Comment:
yeah that's quite funny...  `pygame.surfarray.array3d` needs numpy.. I will add a comment about that ;)